internal/crossdock/server/yarpc/server.go

```diff
@@ -97,4 +97,8 @@ func register(reg *yarpc.Dispatcher) {
 	reg.Register(raw.Procedure("waitfortimeout/raw", WaitForTimeoutRaw))
 
 	reg.Register(crossdockpb.BuildEchoYARPCProcedures(EchoProtobuf{}))
+
+	rawProcedure2 := raw.Procedure("echo/raw2", EchoRaw)
```

Comment:
let's remove these crossdock changes and keep the unit test below!

Reply:
removed.